src/grelu/model/models.py

```diff
@@ -518,17 +518,19 @@ def __init__(
         pos_dropout: float = 0.01,
         attn_dropout: float = 0.05,
         ff_dropout: float = 0.2,
-        norm_kwargs: Optional[dict] = {"eps" : 0.001},
+        norm_kwargs: Optional[dict] = None,
         n_heads: int = 8,
         n_pos_features: int = 32,
         # Head
         crop_len: int = 16,
+        act_func: str = "gelu_borzoi",
         final_act_func: Optional[str] = None,
         final_pool_func: Optional[str] = "avg",
         flash_attn=False,
         dtype=None,
         device=None,
     ) -> None:
+        norm_kwargs = norm_kwargs or {"eps": 0.001}
```

Comment:
This is a very strange method of setting a default value to me. It also has the disadvantage that if the user looks at the __init__ defaults they will not see this and may not realize that this default exists. Why not stick with the earlier approach of "norm_kwargs: Optional[dict] = {"eps" : 0.001}"?

Reply:
https://www.pullrequest.com/blog/python-pitfalls-the-perils-of-using-lists-and-dicts-as-default-arguments/

Reply:
The use of mutable defaults causes a lot of bugs, so it is highly discouraged.

Reply:
I see, thanks for the clarification. In that case, can we mention in the documentation that {"eps" : 0.001} will be added by default?

Reply:
Docs updated